src/iotjs_util.c

```diff
@@ -43,28 +43,29 @@ iotjs_string_t iotjs_file_read(const char* path) {
     ptr = buffer + read;
   }
 #else
-  size_t read = fread(buffer, 1, len, file);
+  size_t read = fread(buffer, 1, (size_t)len, file);
 #endif
   IOTJS_ASSERT(read == (size_t)len);
 
   *(buffer + len) = 0;
 
   fclose(file);
 
-  iotjs_string_t contents = iotjs_string_create_with_buffer(buffer, len);
+  iotjs_string_t contents =
+      iotjs_string_create_with_buffer(buffer, (uint32_t)len);
```

Comment:
The return value of `ftell` is long and there is an assert for checking the returned value is a positive number. It is only safe cast after that. We can declare a new variable for the return value if you think it's better.
E.g:
```c
long ftell_ret = ftell(file);
IOTJS_ASSERT(ftell_ret >= 0);
size_t len = (size_t)ftell_ret;

...

iotjs_string_t contents =
    iotjs_string_create_with_buffer(buffer, len);
```

BTW, I was thinking on that we might check the return value of `ftell` in release mode too.